spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/config/EncryptionAutoConfiguration.java

```diff
@@ -44,6 +45,9 @@
 @AutoConfigureAfter(DefaultTextEncryptionAutoConfiguration.class)
 public class EncryptionAutoConfiguration {
 
+	@Value("${spring.cloud.config.server.encrypt.prefixInvalidProperties:${spring.cloud.config.server.encrypt.prefix-invalid-properties:true}}")
```

Comment:
I did not like doing it this way but if I were to initialize `ConfigServerProperties` in this class it would cause some issues when the config server uses bootstrap.  It looks like certain properties do not get initialized properly and promoted to the main context.  This was the lest amount of disruptive changes to make this work.